### DISCUSSION TITLE
port 4000 and no new data

### DISCUSSION THREAD

**Discussion by dsnowdy** [teslamate-org/teslamate#2551] [General]
Occasionally my Teslamate install stops displaying new data and also port 4000 isn't visible.  Usually, the solution I've found in forums: changing the version from latest to a working version.  I've tried latest, 1.26.1 and 1.25.1.  Currently none of those changes in docker get it working again.  Thanks.  Dave

**Comment by dsnowdy** [comment]
I think it works with 1.25.0.  I can see port 4000 now.  Waiting on data.

**Reply by cwanja** [reply]
Are you running a Raspberry Pi? On Buster? See here: 
https://github.com/adriankumpf/teslamate/issues/2302

**Reply by dsnowdy** [reply]
Thanks for the link.
Pi.  It's working now with using 1.25.0.  Don't know why it breaks from time to time and I have to keep adjusting the version.

Dave

**Reply by cwanja** [reply]
Here is a more expansion comment. #2302 will give you a good overview. Adrian's comment specifically here https://github.com/adriankumpf/teslamate/issues/2302#issuecomment-1011392890 

Couple of options:
1. Downgrade back to 1.25.0 
2. Backport the necessary files, learn more here https://github.com/adriankumpf/teslamate/issues/2302#issuecomment-1011362938
3. Upgrade to Bullseye.

Just be aware that you will need to either upgrade your OS or backport the libraries _eventually_. You can stay on 1.25 forever if you wish, but TeslaMate is not going to work unless you address 2 or 3 above. TeslaMate core was upgraded, so it’s not like a future fix is coming in an update.

Hope it helps!

**Reply by dsnowdy** [reply]
Thanks.  I have been seeing mentions of something like that.  Tried a full upgrade but it didn't go further than it already was.
Will do.
Thanks again.
Dave

**Comment by dsnowdy** [comment]
Thanks again.  I'm now on bullseye with "latest" and it's working. :)

Dave
